### PR TITLE
updated for Zewo 0.14.0

### PR DESCRIPTION
**Pull request by vovasty**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -3,8 +3,8 @@ import PackageDescription
 let package = Package(
     name: "XML",
     dependencies: [
-        .Package(url: "https://github.com/open-swift/C7.git", majorVersion: 0, minor: 10),
-        .Package(url: "https://github.com/VeniceX/File.git", majorVersion: 0, minor: 10),
+        .Package(url: "https://github.com/open-swift/C7.git", majorVersion: 0, minor: 12),
```
Comment (Danappelxx) [inline Package.swift]: We don't use C7 anymore, you can cut this dependency.

File: Sources/XML/XMLNode.swift [hunk 2]
```diff
@@ -287,9 +287,11 @@ public class XMLNode {
         }
         
         xPathContext.pointee.node = self.xmlNode
-
+
```
Comment (Danappelxx) [inline Sources/XML/XMLNode.swift]: cut whitespace

**Review by Danappelxx — CHANGES_REQUESTED** [review]
Great! Just a couple things and we're good to go 👍

**Comment by vovasty** [comment]
done

File: Package.swift (hunk 1 above)
Comment (Danappelxx) [reply Package.swift]: still there :/

**Comment by Danappelxx** [comment]
CI is failing cause of some imports. Fix that and this is good to go!

**Review by Danappelxx — APPROVED** [review]

**Comment by vovasty** [comment]
tests will fail cause needs test data files in Tests/XMLTests/TestData

**Comment by Danappelxx** [comment]
Thats very strange, but you're right. Reverting my commit & merging.

**Pull request merged**